upstream/CVE-2022-3162.json

```
@@ -0,0 +1,62 @@
+{
+  "id": "CVE-2022-3162",
+  "modified": "2022-11-08T21:33:07Z",
+  "published": "2022-11-08T21:33:07Z",
+  "summary": "Unauthorized read of Custom Resources",
+  "details": "Users authorized to list or watch one type of namespaced custom resource cluster-wide can read custom resources of a different type in the same API group without authorization. Clusters are impacted by this vulnerability if all of the following are true: 1. There are 2+ CustomResourceDefinitions sharing the same API group 2. Users have cluster-wide list or watch authorization on one of those custom resources. 3. The same users are not authorized to read another custom resource in the same API group.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "1.22.0"
```

ref. https://github.com/kubernetes/kubernetes/issues/113756#issuecomment-1335667380
```suggestion
              "introduced": "0"
```

---

changed in https://github.com/aquasecurity/vuln-list-k8s/pull/89/commits/ef50ed7d6f25d5159df77d71fbcf7d3d6c7cef27